### PR TITLE
Add requires to interfaces that reference types or attributes without…

### PR DESCRIPTION
**Pull request by dburgener**
… requiring them

### CODE REVIEW

**Comment by dburgener** [comment]
I've updated to add the Signed-off-by.  Thanks!

File: policy/modules/services/procmail.if [hunk 1]
```diff
@@ -90,7 +90,7 @@ interface(`procmail_read_home_files',`
 #
 interface(`procmail_relabel_home_files',`
 	gen_require(`
-		type ppp_home_t;
+		type ppp_home_t, procmail_home_t;
```
Comment (pebenito) [inline policy/modules/services/procmail.if]: In this case it should drop `ppp_home_t` as unused.

File: policy/modules/services/xserver.if [hunk 2]
```diff
@@ -22,6 +22,7 @@ interface(`xserver_restricted_role',`
 		type user_fonts_t, user_fonts_cache_t, user_fonts_config_t;
 		type iceauth_t, iceauth_exec_t, iceauth_home_t;
 		type xauth_t, xauth_exec_t, xauth_home_t;
+		type xdm_t, xdm_tmp_t;
```
Comment (pebenito) [inline policy/modules/services/xserver.if]: `xserver_exec_t` appears to be unused.

File: policy/modules/services/zabbix.if [hunk 3]
```diff
@@ -109,7 +109,7 @@ interface(`zabbix_read_pid_files',`
 #
 interface(`zabbix_agent_tcp_connect',`
 	gen_require(`
-		type zabbix_agent_t;
+		type zabbix_t, zabbix_agent_t;
```
Comment (pebenito) [inline policy/modules/services/zabbix.if]: Seems that `zappix_agent_t` should be removed.

**Pull request merged**